src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -124,6 +124,12 @@
       <Versions>10.10;10.11;10.12;10.13;10.14;10.15</Versions>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="osx">
```

Comment:
Is there a specific reason behind creating a new Item ? cant we just add it to the existing osx item ?

Reply:
11 shall run on arm64 as well. existing versions are x64 only. I followed some other examples.